core/src/main/java/org/jdbi/v3/core/mapper/EnumByOrdinalMapperFactory.java

```diff
@@ -13,21 +13,21 @@
  */
 package org.jdbi.v3.core.mapper;
 
-import static org.jdbi.v3.core.generic.GenericTypes.getErasedType;
-
 import java.lang.reflect.Type;
 import java.util.Optional;
-
 import org.jdbi.v3.core.config.ConfigRegistry;
 
+import static org.jdbi.v3.core.generic.GenericTypes.getErasedType;
+
 /**
- * Produces enum column mappers, which map enums from numeric columns according to ordinal value.
+ * Produces enum column mappers, which map enums from numeric columns according to {@link Enum#ordinal()}.
```

Comment:
Would be nice to add a `{@link}` to the new config property

Reply:
After we decide on the location of said new preference